std/file.d

```diff
@@ -3142,7 +3142,7 @@ private struct DirIteratorImpl
 
         bool stepIn(string directory)
         {
-            auto h = cenforce(opendir(directory.tempCString()), directory);
+            auto h = cenforce(opendir(directory.length ? directory.tempCString() : "."), directory);
```

Comment:
It should not. But I'd play it safe b/c of dmd's dtor bugs - add a separate line to get C-string.